src/Compilers/CSharp/Portable/Symbols/ConstraintsHelper.cs

```diff
@@ -438,6 +488,8 @@ private static bool CheckConstraintsSingleType(TypeSymbol type, CheckConstraints
             Location location,
             DiagnosticBag diagnostics)
         {
+            type = (NamedTypeSymbol)type.TupleUnderlyingTypeOrSelf();
```

Comment:
Which test do you mean?
If you mean `LongTupleConstraints`, I didn't think it would hit this code path. Can you confirm?

---
In reply to: [92736074](https://github.com/dotnet/roslyn/pull/15641#discussion_r92736074) [](ancestors = 92736074)